resolvers/webpack/index.js

```diff
@@ -57,7 +57,10 @@ exports.resolve = function (source, file, settings) {
       // see if we've got an absolute path
       if (!configPath || !isAbsolute(configPath)) {
         // if not, find ancestral package.json and use its directory as base for the path
-        packageDir = findRoot(path.resolve(file))
+        packageDir = findRoot(path.resolve(file), function(dir) {
```

Comment:
please always add a space after the `function` keyword

Reply:
Sure, fixed here, but that really should be enforced by eslint or prettier being set up. As a newcomer to codebase I have no idea what style rules does it follow, especially if the project is not consistent about them. I've found 55 occurrences of `function(` in most important directories of this package, where 7 of them were **in** the very file I've edited.

Reply:
totally agree that we need stricter linting rules in this repo :-)